kayoumido/HEIGVD-PRO-A-11
# V1/140 get role on api

# How to test
* Log with a user
* try to retrieve data for presentations
  * `user/{user}/presentations` : to show all presentations related to the user and your role on that presentations
  * `presentation/{presentation}`: to check a specific presentation
  * a route you want that returns a presentation
